tox.ini

```diff
@@ -18,10 +18,11 @@ deps=
 deps=
 	-rtest-requirements.txt
 	black
-	pylint
+	pylint==2.5.3
+	astroid==2.4.2
 commands=
 	black --check .
-	sh -c 'pylint src; test $(( $? & (1|2|4|32) )) = 0'
+	sh -c 'pylint pushcollector; test $(( $? & (1|2|4|32) )) = 0'
```

Comment:
Hmm, does this work actually?  Wasn't "src" correct since that's where the source code is located?

Reply:
Thanks.